mappings/net/minecraft/block/Block.mapping

```diff
@@ -525,11 +620,23 @@ CLASS net/minecraft/class_197 net/minecraft/block/Block
 		COMMENT @return Whether the block should drop as an item during an explosion
 		ARG 1 explosion
 			COMMENT The explosion that caused the block to break
+	METHOD method_4185 onPlaced (Lnet/minecraft/class_1150;IIIIFFFI)I
+		ARG 1 world
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
 	METHOD method_4187 isFire ()Z
 	METHOD method_4188 isFlowerPot ()Z
 	METHOD method_4189 getMinX ()D
 		COMMENT Returns the block's bounding box' minimum X value
 		COMMENT @return Minimum X value
+	METHOD method_419 onBlockPlaced (Lnet/minecraft/class_1150;IIILnet/minecraft/class_1699;Lnet/minecraft/class_1071;)V
```

Comment:
I mean, it's really not, here's the EndPortalFrameBlock's onPlaced, ![abc](https://i.imgur.com/s9IMI0f.png)

This is where the code that determines the endportal frame's rotation is, and thats exactly what this would do... What would you think that it can do?

Reply:
there's also an onPlaced